src/screens/FeedScreen/hooks/useContentPoll.js

```diff
@@ -37,7 +37,8 @@ const useContentPoll = ({polls, voteCount, isAlreadyPolling: isAlreadyPollingPro
     newItem.isalreadypolling = true;
     newItem.refreshtoken = new Date().valueOf();
     if (multipleChoiceSelected.length === 0) {
-      setIsAlreadyPolling(false);
+      setIsAlreadyPolling(true);
+      inputSingleChoicePoll(polls[0].polling_id, NO_POLL_UUID).catch((e) => console.log(e));
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `setIsAlreadyPolling` function is now called with `true` even when `multipleChoiceSelected.length === 0`. This could potentially lead to incorrect state if there are no polls selected. Also, error handling for `inputSingleChoicePoll` only logs the error to the console. It would be better to handle the error in a more user-friendly way, perhaps by setting an error state and displaying a message to the user.
```diff
- setIsAlreadyPolling(false);
+ setIsAlreadyPolling(true);
+ inputSingleChoicePoll(polls[0].polling_id, NO_POLL_UUID).catch((e) => console.log(e));
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->